docs/ja/src/reference/apps/joint_position_sender.md

```diff
@@ -0,0 +1,21 @@
+# openrr_apps_joint_position_sender
+
+`openrr_apps_joint_position_sender`ではマニピュレータなど関節ロボットについての各関節の値を指定することができます。
+
+```bash
+openrr_apps_joint_position_sender
+```
+
+![joint_pose_sender](image/joint_pose_sender.png)
+
+## 使い方
+
+```bash
+openrr_apps_joint_position_sender [OPTIONS]
+```
+
+| Options                         | Description                                        |
+| ------------------------------- | -------------------------------------------------- |
+| -c, --config-path <CONFIG_PATH> | 設定ファイルへのパス                               |
+| --config <CONFIG>               | デフォルトまたは設定ファイルで指定した設定の上書き |
```

Comment:
```suggestion
| --config \<CONFIG>               | デフォルトまたは設定ファイルで指定した設定の上書き |
```

`<CONFIG>` will be handled as html, and not displayed as `<CONFIG>` in markdown.

<img width="240" alt="config" src="https://user-images.githubusercontent.com/43724913/204219022-f6a51dbf-c319-4bcd-9816-acf234890e32.png">